Cartfile

```diff
@@ -1,2 +1,2 @@
 github "bitmark-inc/tweetnacl-swiftwrap"
-github "kishikawakatsumi/KeychainAccess"
+github "kishikawakatsumi/KeychainAccess" == 3.1.2
```

Comment:
This will make the dependency stick with this version and cannot support Swift 5. I would suggest to not do that.